flow-typed/custom/jsdom.js

```diff
@@ -0,0 +1,332 @@
+    HTMLQuoteElement: typeof HTMLQuoteElement,
+    HTMLScriptElement: typeof HTMLScriptElement,
+    HTMLSelectElement: typeof HTMLSelectElement,
+    HTMLSourceElement: typeof HTMLSourceElement,
+    HTMLSpanElement: typeof HTMLSpanElement,
+    HTMLStyleElement: typeof HTMLStyleElement,
+    HTMLTableCaptionElement: typeof HTMLTableCaptionElement,
+    HTMLTableCellElement: typeof HTMLTableCellElement,
+    HTMLTableColElement: typeof HTMLTableColElement,
+    HTMLTableElement: typeof HTMLTableElement,
+    HTMLTimeElement: typeof HTMLTimeElement,
+    HTMLTitleElement: typeof HTMLTitleElement,
+    HTMLTableRowElement: typeof HTMLTableRowElement,
+    HTMLTableSectionElement: typeof HTMLTableSectionElement,
+    HTMLTemplateElement: typeof HTMLTemplateElement,
+    HTMLTextAreaElement: typeof HTMLTextAreaElement,
+    HTMLTrackElement: typeof HTMLTrackElement,
+    HTMLUListElement: typeof HTMLUListElement,
+    HTMLUnknownElement: typeof HTMLUnknownElement,
+    HTMLVideoElement: typeof HTMLVideoElement,
+    StyleSheet: typeof StyleSheet,
+    MediaList: typeof MediaList,
+    CSSStyleSheet: typeof CSSStyleSheet,
+    CSSRule: typeof CSSRule,
+    CSSStyleRule: typeof CSSStyleRule,
+    CSSMediaRule: typeof CSSMediaRule,
+    CSSImportRule: typeof CSSImportRule,
+    CSSStyleDeclaration: typeof CSSStyleDeclaration,
+    StyleSheetList: typeof StyleSheetList,
+    XPathExpression: typeof XPathExpression,
+    XPathResult: typeof XPathResult,
+    XPathEvaluator: typeof XPathEvaluator,
+    NodeFilter: typeof NodeFilter,
+    close: () => void,
+    [key: string]: mixed,
+    ...
+};
+
+declare type jsdom$BinaryData =
+    | ArrayBuffer
+    | DataView
+    | Int8Array
+    | Uint8Array
+    | Uint8ClampedArray
+    | Int16Array
+    | Uint16Array
+    | Int32Array
+    | Uint32Array
+    | Float32Array
+    | Float64Array;
+
+declare class jsdom$VirtualConsole mixins events$EventEmitter {
+    on<K: $Keys<Console>>(method: K, callback: $ElementType<Console, K>): this;
+    on(event: "jsdomError", callback: (e: Error) => void): this;
+    sendTo(console: Console, options?: jsdom$VirtualConsoleSendToOptions): this;
+}
+
+declare interface jsdom$VirtualConsoleSendToOptions {
+    omitJSDOMErrors: boolean;
+}
+
+declare class jsdom$CookieJar extends tough$CookieJar {}
+
+declare interface jsdom$ReconfigureSettings {
+    windowTop?: jsdom$DOMWindow;
+    url?: string;
+}
+
+declare interface jsdom$FetchOptions {
+    cookieJar?: jsdom$CookieJar;
+    referrer?: string;
+    accept?: string;
+    element?:
+        | HTMLScriptElement
+        | HTMLLinkElement
+        | HTMLIFrameElement
+        | HTMLImageElement;
+}
+
+declare interface jsdom$ResourceLoaderConstructorOptions {
+    strictSSL?: boolean;
+    proxy?: string;
+    userAgent?: string;
+}
+
+declare class jsdom$ResourceLoader {
+    constructor(obj?: jsdom$ResourceLoaderConstructorOptions): this;
+    fetch(url: string, options?: jsdom$FetchOptions): ?Promise<Buffer>;
```

Comment:
Unlike the `react-render-server` version, this makes `options` optional. I checked the actual JSDOM code, and this is the case.